packages/components/src/spinner/style.scss

```diff
@@ -0,0 +1,44 @@
+.components-spinner--svg {
+	animation: 2s linear infinite both rotatesvg;
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+
+@keyframes rotatesvg {
+	0% {
+		transform: rotateZ(0deg);
+	}
+	100% {
+		transform: rotateZ(360deg);
+	}
+}
+
+.components-spinner--circle {
```

Comment:
If I understand correctly, this is the blue part of the spinner that in this PR contracts and expands, so it's more of a quarter-circle. I realize this is nitpick territory, but I think it'd help readability. How about something like `.components-spinner__indicator`? Note the __ instead of --.